website/app/components/doc/page/header/search/index.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+import { autocomplete } from '@algolia/autocomplete-js';
+
+import { getItemsFunction } from './parts/getItemsFunction';
+import { templatesHeaderFunction } from './parts/templatesHeaderFunction';
+// import { templatesFooterFunction } from './parts/templatesFooterFunction';
```

Comment:
leaving this for when we will add a "show-all" link at the end of a list of results

Reply:
What will the "show-all" link do? Take users to a page listing the search results?

Reply:
I don't know yet, likely will increase the limit from 5 results to infinite for that specific type of content. 
In any case, it will be done in https://hashicorp.atlassian.net/browse/HDS-2888